### PR TITLE
Fi bo 131 add missing use case description

### PR DESCRIPTION
**Pull request by Cebox82**

### CODE REVIEW

File: docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+# 2 Flow of Events
+
+## 2.1 Basic Flow
+
+- User clicks on dashboard
+- User selects a specific time slot
+  Data gets checked for availability depending on the time period
+- If more than one cash flow of a category is available, the value is summed up
```
Comment (Jonas-Karl) [inline docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md:22]:
```suggestion
- If more than one cashflow of a category is available, the value is summed up
```

File: docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+### 2.1.3 Narrative
+
+```gherkin
+Feature: show data in dashboard
+
+  As a signed-in user
+  I want to display the summed cash flows 
+  in a pie chart 
+  depending on a time window and category
```
Comment (Jonas-Karl) [inline docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md:45]:
```suggestion
  depending on a time window and seperated by category
```

its not depending on a category but the cashflows are seperated by category

File: docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+  Background:
+    And I am on the dashboard
+
+  Scenario: show cash flow data depending on time slot
```
Comment (Jonas-Karl) [inline docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md:50]:
```suggestion
  Scenario: show cashflow data depending on time slot
```

File: docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    Given I am signed in with username "USER" and password "PASSWORD"
+    And I am on the "dashboard" page
+    When I select a specific time slot
```
Comment (Jonas-Karl) [inline docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md:53]: Should we also add the scenario when the user does not select any specific timeslot?

In this case we should mentioned that the standart value is the complete time from the first to the last cashflow entry

File: docs/SRS.md [hunk 5]
```diff
@@ -128,22 +128,32 @@ Until the end of December we plan to implement:
...
-#### 3.1.3 Logging out
+[Logging in](./use_cases/UC_2_login_account/UC_2_login_account.md)
 
-In case you share your phone, have multiple accounts or just want to be cautious about your privacy you should be able to manually log out.
+
+#### 3.1.3 Change added data
+
+In case you want to change data in the cashflows, you should be able to edit the data with a simple click on the respective item.
+
+[Change added data](./use_cases/UC_3_change_added_data/UC_3_change_added_data.md)
```
Comment (Jonas-Karl) [inline docs/SRS.md:156]: In the UC_3 there is a Use Case Diagram that reflects the register szenario. If we have the draw.io / svg files for the change_added_data scenario already, do you mind to exchange them?

**Review by Jonas-Karl — CHANGES_REQUESTED** [review]
Pretty cool mockups!
I only found some typos and added some comments here and there. Would be cool if you can also fix the error in UC3

File: docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md (hunk 4 above)
Comment (Cebox82) [reply docs/use_cases/UC_5_show_data_in_dashboard/UC_5_show_data_in_dashboard.md:53]: I think its a good idea

**Review by Jonas-Karl — APPROVED** [review]

**Pull request merged**